### PR TITLE
New script sarcasm.py: Adds random capitalization

### PR DESCRIPTION
**Pull request by fsaev**
## Script info

- Script name: Sarcasm.py
- Version: 1.0

- Requirements:

- Min WeeChat version: 

- Script tags: display py3

## Description

Adds random capitalization to text to indicate sArCAsm

## Checklist (new script)

- [x] Single commit, single file added
- [x] Commit message: `New script name.py: short description…`
- [x] No similar script already exists
- [x] Name: max 32 chars, only lower case letters, digits and underscores
- [x] Unique name, does not already exist in repository
- [x] No shebang on the first line
- [x] Comment in script with name/pseudo, e-mail and license
- [x] Only English in code/comments
- [x] Pure WeeChat API used, no extra API
- [x] Function `hook_process` is used for any blocking call
- [x] For Python script: works with Python 3 (Python 2 support is optional)
- [x] Score 100 / 100 displayed by [weechat-script-lint](https://github.com/weechat/weechat-script-lint)

### CODE REVIEW

File: python/sarcasm.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        else:
+            newstring += arg.lower()
+
+    weechat.command(buffer, newstring)
+
+    return weechat.WEECHAT_RC_OK
+
+if __name__ == "__main__" and import_ok:
+    if weechat.register(SCRIPT_NAME, SCRIPT_AUTHOR, SCRIPT_VERSION, SCRIPT_LICENSE, SCRIPT_DESC, "", ""):
+        weechat_version = weechat.info_get("version_number", "") or 0
```
Comment (flashcode) [inline python/sarcasm.py:40]: This variable and its initialization are not used, they can be safely removed.

File: python/sarcasm.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Script Name: sarcasm.py
+# Script Author: Fsaev
+# Script License: GPLv3
+
+SCRIPT_NAME = 'sarcasm.py'
```
Comment (flashcode) [inline python/sarcasm.py:7]: By convention the extension should not be part of the script name:

```suggestion
SCRIPT_NAME = 'sarcasm'
```

File: python/sarcasm.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```
Comment (flashcode) [inline python/sarcasm.py:1]: UTF-8 encoding is default in Python 3, so this line is not necessary.

**Review by flashcode — CHANGES_REQUESTED** [review]

**Comment by fsaev** [comment]
All fixed! Do you want me to squash it to a single commit as well?

**Comment by flashcode** [comment]
Yes please squash in a single commit, thanks!

**Review by flashcode — APPROVED** [review]

**Comment by flashcode** [comment]
Thanks, merged!

**Pull request merged**